system/HTTP/Response.php

```diff
@@ -257,6 +270,10 @@ public function __construct($config)
 		$this->cookieHTTPOnly = $config->cookieHTTPOnly;
 		$this->cookieSameSite = $config->cookieSameSite ?? $this->cookieSameSite;
 
+    // CORS
+    $this->CORSEnabled     = $config->CORSEnabled;
```

Comment:
this is bc break without `??` check, look at previous config set:

```php
$this->CORSEnabled = $config->CORSEnabled ?? $this->CORSEnabled;
```

Reply:
this is not form my code ...

Reply:
It's not necessary, all other config params are mandatories in **App/Config/App.php** file

Reply:
when user run `composer update` with existing their appstarter, that new config will make application break. That's why need check if that exists.

Reply:
ok, I understand and I'll fix it in my PR but just looking in **Systema/HTTP/Response.php** file we have this:
>		$this->CSPEnabled     = $config->CSPEnabled;
>		$this->cookiePrefix   = $config->cookiePrefix;
>		$this->cookieDomain   = $config->cookieDomain;
>		$this->cookiePath     = $config->cookiePath;
>		$this->cookieSecure   = $config->cookieSecure;
>		$this->cookieHTTPOnly = $config->cookieHTTPOnly;
>		$this->cookieSameSite = $config->cookieSameSite ?? $this->cookieSameSite;

just one are using this and it's in "*production*"

Reply:
The thing is the last setting is just new after v4.0.4, so users using v4.0.4 and below do not have the sameSite setting in their App config. Adding the `??` will ensure fallback in case the App config is not updated to have the sameSite setting.

This case also apply to your PR. This is a new addition so existing users will be lacking that.

Reply:
it's done ..